Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,28 @@
+# Principles
+
```

Comment:
TODO: Reference this against the ZAC principles and OMP principles (Finalizing next week ) and LF principles.